doc/tutorial_sector.rst

```diff
@@ -0,0 +1,637 @@
+        4 -> 40
+        10 -> 41
+        31 -> 41
+        42 -> 41
+        29 -> 43
+        10 -> 45
+    }
+
+
+In the terminal, this will show up as a list of jobs to be run:
+
+.. code:: console
+
+    Building DAG of jobs...
+    Job stats:
+    job                                   count
+    ----------------------------------  -------
+    add_electricity                           1
+    add_export                                1
+    add_extra_components                      1
+    base_network                              1
+    build_base_energy_totals                  1
+    build_base_industry_totals                1
+    build_bus_regions                         1
+    build_clustered_population_layouts        1
+    build_cop_profiles                        1
+    build_demand_profiles                     1
+    build_heat_demand                         1
+    build_industrial_distribution_key         1
+    build_industry_demand                     1
+    build_natura_raster                       1
+    build_osm_network                         1
+    build_population_layouts                  1
+    build_powerplants                         1
+    build_renewable_profiles                  5
+    build_shapes                              1
+    build_ship_profile                        1
+    build_solar_thermal_profiles              1
+    build_temperature_profiles                1
+    clean_osm_data                            1
+    cluster_network                           1
+    copy_config                               1
+    download_osm_data                         1
+    override_respot                           1
+    prepare_airports                          1
+    prepare_energy_totals                     1
+    prepare_gas_network                       1
+    prepare_heat_data                         1
+    prepare_network                           1
+    prepare_ports                             1
+    prepare_sector_network                    1
+    prepare_transport_data                    1
+    prepare_transport_data_input              1
+    prepare_urban_percent                     1
+    retrieve_cost_data                        1
+    retrieve_databundle_light                 1
+    simplify_network                          1
+    solve_sector_network                      1
+    solve_sector_networks                     1
+    total                                    46
+
+
+
+
+Myopic Foresight Scenarios
+=============================================
+
+
+Configuration
+---------------------------------------------
+
+The configuration to run the tutorial for the myopic foresight scenario is present
+in the ``test/config.test_myopic.yaml`` file.
+
+.. code:: yaml
+
+    foresight: myopic
+
+Scenarios can be defined like for electricity-only studies, but with additional
+wildcard options. For the myopic foresight mode, the ``{planning_horizons}`` wildcard
+defines the sequence of investment horizons.
```

Comment:
If I understand properly, the current implementation allows to run the myopic optimisation only for the sectoral model. If that's a correct understanding, can we add a comment on that also to the tutorial?

Reply:
Super :)